refinery/tool_manager/tests.py

```diff
@@ -515,37 +515,44 @@ def setUp(self):
             tool_annotation["workflow_engine_uuid"] = self.workflow_engine.uuid
             create_tool_definition(tool_annotation)
 
+        self.dataset = create_dataset_with_necessary_models()
+        self.dataset.set_owner(self.user)
+
+        self.public_dataset = create_dataset_with_necessary_models()
+        self.public_dataset.share(self.public_group)
+
         # Make reusable requests & responses
-        self.get_request = self.factory.get(self.tool_defs_url_root)
+        self.get_request = self.factory.get(
+            "{}?dataSetUuid={}".format(
+                self.tool_defs_url_root,
+                self.dataset.uuid
+            )
+        )
         force_authenticate(self.get_request, self.user)
         self.get_response = self.tool_defs_view(self.get_request)
 
         self.tool_json = self.get_response.data[0]
 
         self.delete_request = self.factory.delete(
             urljoin(self.tool_defs_url_root, self.tool_json['uuid']))
-        force_authenticate(self.delete_request, self.user)
         self.delete_response = self.tool_defs_view(self.delete_request)
```

Comment:
The methods besides `GET` aren't allowed so forcing auth on them here was overkill since the `'Method "xxxx" not allowed.' is returned before auth is being checked.